CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+#
+# CMAKE SETUP (version, build style)
+#
+cmake_minimum_required (VERSION 2.8 FATAL_ERROR)
+project ("Quick-DER" C)
+
+set (CMAKE_MACOSX_RPATH 0)  # Don't use rpaths (but we don't build executables anyway)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR}/cmake)
+
+include(MacroEnsureOutOfSourceBuild)
+macro_ensure_out_of_source_build("Do not build Quick-DER in the source directory.")
+
+include(MacroAddTestBuilder)
+
+#
+# OPTIONS / BUILD SETTINGS
+#
+option (DEBUG
+	"Produce verbose output while unpcaking and packing DER"
+	ON)
+option (SPEC_RFC
+	"Build include files from ASN.1 fragments of RFCs"
+	ON)
+option (SPEC_ITU
+	"Build include files from ASN.1 fragments of ITU specs"
+	ON)
+option (SPEC_ARPA2
+	"Build include files from ASN.1 specs for ARPA2 projects"
+	ON)
+
+set(Quick-DER_MAJOR_VERSION 0)
+set(Quick-DER_MINOR_VERSION 1)
+set(Quick-DER_PATCH_VERSION 5)  # RC5, will move to 0.2 for release
```

Comment:
Quick-DER is al voorbij 1.0 joh!  Best een erg stabiel ding [al was het dat nog vooral voor EXPLICIT TAGS].

Aschwin heeft een truc waarbij hij git raadpleegt om de dichtstbijzijnde versie te versieren met logs, misschien moeten we dat hier in zien te krijgen als autogen-versioning.